frontend/packages/console-app/src/actions/providers/default-provider.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
```

Comment:
@bipuladh This `react` namespace import isn't needed anymore, since we switched from `"jsx": "react"` to `"jsx": "react-jsx"` - see #14864 for details.

@logonoff @rhamilto Should we add some ESLint rule that disallows such `react` namespace imports for the sake of consistency?

Reply:
At least for console, there already is one:

![Uploading image.png…]()

Reply:
I've just realized this file uses `React.useMemo` so the `react` namespace import is indeed necessary here.

Sorry for confusion, please discard my comment above :confounded: